src/pages/help-centre/index.tsx

```diff
@@ -40,3 +39,5 @@ const HelpCentre = () => {
 }
 
 export default WithIntl()(HelpCentre)
+
+export const Head = () => <script type="application/ld+json">{JSON.stringify(faq_schema)}</script>
```

Comment:
Could you please share why this HOC exist?

Reply:
This is the new way of using scripts in gatsby, we are using Gatsby head api instead of react-helmet
Please checkout this: https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-head/